### ISSUE TITLE
Linux standalone app build not available at the moment

### ISSUE THREAD

**Issue by gdurif** [diyabc/diyabcGUI#18]
Problem with `electricShine`

https://github.com/chasemc/electricShine/issues/294

TODO: write a fix for electricShine

**Comment by gdurif** [comment]
For the moment, Linux users will have to use the R package.

**Comment by gdurif** [comment]
Relevant to generate a portable version of R for linux users ?

**Comment by fradav** [comment]
Flatpak or snap?

**Comment by gdurif** [comment]
My opinion is to forget the standalone for Linux. If users can install flatpak or snap apps, they can install R and directly use the R package.

**Comment by fradav** [comment]
I agree, but snap/flatpak are preinstalled on some distros (Ubuntu, gnome etc.). There is also the appimage format which doesn't require anything installed. 

At this point this should be a very low priority, and mostly CI dedicated task.

**Comment by fradav** [comment]
perhaps a container based approach (docker/singularity) is more relevant for some use case (headless server with the shiny app server) ?

**Comment by gdurif** [comment]
Regarding the shiny app server, the "difficult" part is the setup of the shiny server, which is detailed https://docs.rstudio.com/shiny-server/ (a docker container seems to even be available https://hub.docker.com/r/rocker/shiny).

Once the server is up, setting our app is quite simple, c.f. https://github.com/diyabc/diyabcGUI#shiny-server-installation, and should be accessible for anyone able to manage the shiny server setup part.

**Comment by gdurif** [comment]
Anyway, I think I will not have time for this any time soon, so maybe in the future.
